apiary-metastore-events/apiary-hive-events/src/test/java/com/expediagroup/apiary/extensions/events/metastore/event/ApiaryListenerEventFactoryTest.java

```diff
@@ -60,20 +55,11 @@ public void init() {
   private <T extends ListenerEvent> T mockEvent(Class<T> clazz) {
     T event = mock(clazz);
     when(event.getStatus()).thenReturn(true);
-    doAnswer(new Answer<Void>() {
```

Comment:
discussed we can't test new logic due final methods in abstract class mockito is throwing errors even newer versions (3.x, 4.x)